tests/test_pbkm_workflow_validation.ps1

```
@@ -0,0 +1,196 @@
+Write-Host "`n[Part 2] Simulating PBKM pfsshell Workflows on Virtual Disk Fixture..." -ForegroundColor Yellow
+
+if (Test-Path $WorkingDir) { Remove-Item -Path $WorkingDir -Recurse -Force }
+New-Item -ItemType Directory -Path $WorkingDir -Force | Out-Null
+
+$imgFile = Join-Path $WorkingDir "pbkm_sim.img"
+$stream = [System.IO.File]::Create($imgFile)
+$stream.SetLength(8L * 1024L * 1024L * 1024L)
+$stream.Close()
+Write-Host "   Created 8 GiB fixture image: $imgFile"
+
+function Run-Pfsshell {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Run-Pfsshell' uses an unapproved verb.

(PSUseApprovedVerbs)